tests/releases/reference_docs_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package releases
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/crd/crdloader"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/util/repo"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func TestReferenceDoc(t *testing.T) {
+	crds, err := crdloader.LoadAllCRDs()
+	if err != nil {
+		t.Fatalf("error loading crds: %v", err)
+	}
+
+	var missing []string
+	for _, crd := range crds {
+		for _, version := range crd.Spec.Versions {
+			gvk := schema.GroupVersionKind{
+				Group:   crd.Spec.Group,
+				Version: version.Name,
+				Kind:    crd.Spec.Names.Kind,
+			}
+			if gvk.Group == "core.cnrm.cloud.google.com" {
+				continue
+			}
+			if gvk.Group == "customize.core.cnrm.cloud.google.com" {
+				continue
+			}
+			if strings.HasPrefix(gvk.Version, "v1alpha1") {
```

Comment:
OK for now. I think if we want customer's to use alpha resources provisionally we need to document them but with some sort of warning.